### DISCUSSION TITLE
Add transparent foreground image for Ripples panel

### DISCUSSION THREAD

**Discussion by Zackptg5** [rocksdanister/lively#595] [Ideas]
With the underwater background, the included panel foreground images don't really work with it. Having a transparent foreground image for the panel however, looks amazing. I attached one that I whipped up real quick in gimp. I'd love for you to include it in future builds - obviously no copyright crap or whatever cause it's literally an alpha channel and that's about it lol

Here's a screenshot:
![Capture](https://user-images.githubusercontent.com/5107713/124493553-c3c73a00-dd83-11eb-957a-04ac56bd2f58.png)

Here's the picture:
![Transparent](https://user-images.githubusercontent.com/5107713/124493166-50bdc380-dd83-11eb-8c14-17650e992417.png)

**Comment by rocksdanister** [comment]
Yeah that's a good idea.
ps: I prefer grass with underwater 😄

**Reply by Zackptg5** [reply]
grass does look nice :)
